javascript/packages/orchestrator/src/paras.ts

```diff
@@ -88,6 +88,11 @@ export async function generateParachainFiles(
     if (plainData.relay_chain) plainData.relay_chain = relayChainSpec.id;
     if (plainData.genesis.runtime?.parachainInfo?.parachainId)
       plainData.genesis.runtime.parachainInfo.parachainId = parachain.id;
+    if (
```

Comment:
Hi @MucTepDayH16 can we move this logic to `clearAuthorities` in `equilibrium.ts` and decorate that function to remove this custom logic here.

Thanks!!

Reply:
Ok, fixing it